src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -616,8 +616,8 @@ public int getMaximumPacketSize() {
      * This method will return a new array each time it is invoked.
      *
      * @return a non-null, possibly zero-length array of application protocol
-     *         {@code String}s.  The array is ordered based on protocol
-     *         preference, with {@code protocols[0]} being the most preferred.
+     *         {@code String}s.  The array is placed in descending order of
```

Comment:
The phrase "descending order" seems more appropriate for numerical values. I think the previous wording was more clear, with a small change: "The array is ordered based on protocol preference, with the first entry being the most preferred."

Reply:
Hm, much better.  Thanks!

Reply:
> The phrase "descending order" seems more appropriate for numerical values. I think the previous wording was more clear, with a small change: "The array is ordered based on protocol preference, with the first entry being the most preferred."

Thanks Sean - I had the same comment - but for some reason it seemed to have been lost when I clicked on Approve. Not sure what happened to it :-(

Reply:
Thank you!